tests/test_integration_service.py

```diff
@@ -1115,3 +1116,138 @@ def test_detail_list(self):
+                                },
+                                "tilstande": {
+                                    "organisationgyldighed": [
+                                        {
+                                            "gyldighed": "Aktiv",
+                                        }
+                                    ]
+                                },
+                            }
+                        ]
+                    }]
+                ]
+            }
+        )
+
+        self.assertRequestResponse(
+            '/service/o/00000000-0000-0000-0000-000000000000/'
+            'address_autocomplete/?q=42',
+            {
+                'message': 'No local municipality found!',
+                'status': 404,
+            },
+            status_code=404,
+        )
+
+    @freezegun.freeze_time('2016-06-06')
+    @util.mock()
+    def test_autocomplete_invalid_municipality(self, mock):
+        mock.get(
+            'http://mox/organisation/organisation'
+            '?uuid=00000000-0000-0000-0000-000000000000'
+            '&virkningfra=2016-06-06T00%3A00%3A00%2B02%3A00'
+            '&virkningtil=2016-06-07T00%3A00%3A00%2B02%3A00',
+            json={
+                "results": [
+                    [{
+                        "id": "00000000-0000-0000-0000-000000000000",
+                        "registreringer": [
+                            {
+                                "attributter": {
+                                    "organisationegenskaber": [
+                                        {
+                                            "brugervendtnoegle": "bvn",
+                                            "organisationsnavn": "onavn",
+                                        }
+                                    ]
+                                },
+                                "relationer": {
+                                    "myndighed": [
+                                        {
+                                            "urn": "kaflaflibob",
+                                        }
+                                    ]
+                                },
+                                "tilstande": {
+                                    "organisationgyldighed": [
+                                        {
+                                            "gyldighed": "Aktiv",
+                                        }
+                                    ]
+                                },
+                            }
+                        ]
+                    }]
+                ]
+            }
+        )
+
+        self.assertRequestResponse(
+            '/service/o/00000000-0000-0000-0000-000000000000/'
+            'address_autocomplete/?q=42',
+            {
+                'message': 'No local municipality found!',
+                'status': 404,
+            },
+            status_code=404,
+        )
+
+    @patch('mora.service.org.requests')
```

Comment:
`s/mora.service.org//`? Specifying the value after import seems odd. The original module should work quite well in most cases, unless someone's doing something silly — in which case this probably won't work either…

Reply:
I'm not sure what you mean by specifying the value after import.

My idea here is to be as specific as possible in my mocking, as apposed to just blanket-mocking all the uses of `requests`, which could lead to potential problems in our use of `requests` in `lora.py`

Reply:
Oh, I see! For what it's worth I added an `allow_mox` argument to `mock()` in 320b905e2bd768a7e3969f6c4470810bf3180a79 so that you can mock everything other than LoRA. It does the same, but based on URL prefixes instead.

Reply:
I've updated the tests to use `mock()`. The `allow_mox` argument doesn't play well with minimox, it seems, though - but fixing that seemed a bit out of scope for this.